torch/_inductor/codegen/cpp.py

```diff
@@ -1940,6 +1940,8 @@ def codegen_loops(self, code, worksharing):
     @property
     def assert_function(self) -> str:
         if V.graph.aot_mode:
+            # TODO: Using AOTI_TORCH_CHECK is causing performance drop for some models
+            # compared with JIT Inductor which uses TORCH_CHECK
```

Comment:
Create an issue with repro instructions, and then maybe @chenyang78 can take a look.

Reply:
The issue with repro has been submitted: https://github.com/pytorch/pytorch/issues/126665
Since the freezing support requires this PR, in the repro I didn't set the freezing flag and the main branch could also reproduce this issue on these two models: `T5Small` and `T5ForConditionalGeneration`.